autoprotocol/liquid_handle/dispense.py

```diff
@@ -195,35 +236,41 @@ def default_volume_resolution(volume):
 
     # pylint: disable=unused-argument
     @staticmethod
-    def default_prime(volume):
+    def default_prime(volume: Optional[Unit] = None) -> Unit:
         """Default priming volume
 
         Parameters
         ----------
-        volume : Unit
+        volume : Optional[Unit]
 
         Returns
         -------
         Unit
             prime volume
         """
-        return Unit(600, "uL")
+        if volume:
+            return parse_unit(volume, "ul")
+        else:
+            return Unit(600, "uL")
```

Comment:
would it be reasonable to set the default value of `volume=Unit(600, "uL")`?

Reply:
Yup, much cleaner!